src/dbus/interface/mod.rs

```diff
@@ -1,5 +1,12 @@
 pub mod composite_device;
+pub mod force_feedback;
 pub mod manager;
 pub mod performance;
 pub mod source;
 pub mod target;
+
+/// Used to return the name of a DBus interface
```

Comment:
This comment is missing on some of the other uses

Reply:
I will actually be removing this because I discovered that every zbus `Interface` has a `name()` method that does this exact thing.